fhir-persistence-schema/README.md

```diff
@@ -2,6 +2,11 @@
 
 Builds and manages the multi-tenant FHIR R4 RDBMS schema (Db2). Includes Derby support for use in unit tests.
 
+# To Execute the fhir-persistence-schema 
+
+1. Print the schema to files by executing the SchemaPrinter:
```

Comment:
We have the steps for "Printing the schema" below; this should say something like "Execute the fhir-persistence-schema.jar from a terminal and pass the appropriate command:"
Alternatively, you could maybe just remove this line and turn the `java -jar ...` snippet into a shell sample.

Reply:
Addressed, let me know if you want me to fix up fhir_admin